src/stories/Table.js

```diff
@@ -1,4 +1,4 @@
-import * as _ from 'lodash'
+import * as cloneDeep from 'lodash/cloneDeep'
```

Comment:
This was really weird :confused: :alien: :satellite: 
Even though in all the rest js files we do imports like
```
import map from 'lodash/map'
```
in here this didn't work and the storybook was failing.
I guess that this is b/c the storybook uses its own transpilation & bundling configuration which differs from what we use to generate the dist.